takari-lifecycle-plugin/src/main/java/io/takari/maven/plugins/compile/jdt/CompilerJdt.java

```diff
@@ -445,6 +449,13 @@ public void addGeneratedSource(Output<File> generatedSource) {
       sources.put(generatedSource.getResource(), generatedSource);
       processedQueue.add(generatedSource.getResource());
     }
+
+    @Override
+    public void onAnnotationProcessing() {
```

Comment:
This is "on annotation processing" callback and I think `on<ActionType>` is fairly standard callback naming convention.